netboot/main.go

```diff
@@ -139,6 +145,7 @@ func boot(ifname string, dhcp dhcpFunc) error {
 		netconf  *netboot.NetConf
 		bootfile string
 		err      error
+	    client  *http.Client
```

Comment:
have you ran gofmt on this? this doesn't seem aligned well.

Reply:
Done now. Might have had a softtab problem..